Improved & simplified documentation - Doorbird (#11621)
home-assistant/home-assistant.github.io: source/_integrations/doorbird.markdown

## Setup

> It is recommended to setup a new account on your Doorbird App for use with Home Assistant. This can be added via the Doorbird App by clicking settings (cog icon) -> Administration-> LOGIN (using your App Administration details). Under the "USER" section, choose "Add". This new user account requires specific permissions enabled (depending on what functionality you want). Permissions can be found under "Permissions". The following permissions are recommended:

Reviewer: `setup` -> `set up`. That change is invalid. `setup` is a noun, which is arranged already. `set up` is a verb to tell the user to take action.

Please revert that.

```suggestion
It is recommended to set up a new account on your Doorbird App for use with Home Assistant. This can be added via the Doorbird App by clicking settings (cog icon) -> Administration-> LOGIN (using your App Administration details). Under the "USER" section, choose "Add". This new user account requires specific permissions enabled (depending on what functionality you want). Permissions can be found under "Permissions". The following permissions are recommended:
```

Author: Thanks @frenck ... too many years of clicking setup.exe!

- "Watch Always" (live view)
- "History" (last motion)
- "Motion" (last motion)

> - "API-Operator" (this needed to be anabled as a minimum)

Reviewer: ```suggestion
- "API-Operator" (this needed to be enabled as a minimum)
```

#### Schedules

> Once events have been registered on the DoorBird device, they must be attached to a schedule using the official DoorBird app on Android or iOS. Currently there are schedules available for doorbell, motion, relay, and RFID events (on supported Doorbird devices).

Reviewer: ```suggestion
Once events have been registered on the DoorBird device, they must be attached to a schedule using the official DoorBird app on Android or iOS. Currently, there are schedules available for doorbell, motion, relay, and RFID events (on supported Doorbird devices).
```

The schedules can be found by navigating to the following area of the Doorbird app (Android or IOS):

Settings (cog icon) -> Administration -> LOGIN LOGIN (using your App Administration details) -> (under "EXPERT SETTINGS") Schedule for doorbell

- `Push notification`
- `Trigger Relay ("Relay 1" or "Relay 2")`
- `HTTP(S) Calls (button, motion/movement, RFID)`

> Click on the dropdown button in the top left and choose your specific "Schedule for actions" (listed above). Depending on your slection, you may have to click on the center title to see the sub-categories menu. 

Reviewer: ```suggestion
Click on the dropdown button in the top left and choose your specific "Schedule for actions" (listed above). Depending on your selection, you may have to click on the center title to see the sub-categories menu.
```
